business/pom.xml

```diff
@@ -17,6 +17,11 @@
       <artifactId>commons-collections</artifactId>
       <version>3.2.2</version>
     </dependency>
+    <dependency>
```

Comment:
groupId of modules are inherited from the parent pom, so it's still org.mskcc.cbio since we didn't change it. All the modules in a project shares the same groupId.